api/billingd/service/service.go

```diff
@@ -636,6 +636,29 @@ func getUnits(product Product, total int64) (freeUnits, paidUnits int64) {
 	return
 }
 
+func addProductToSummary(summary map[string]interface{}, product Product, total int64) {
+	summary[product.Key+"_name"] = product.Name
+	summary[product.Key+"_units"] = product.Units
+	summary[product.Key+"_free_quota_size"] = product.FreeQuotaSize
+	summary[product.Key+"_total"] = total
+}
+
+func (s *Service) getSummary(cus *Customer, deps int64) map[string]interface{} {
+	summary := map[string]interface{}{
+		"account_status":       cus.AccountStatus(),
+		"billable":             cus.Billable,
+		"delinquent":           cus.Delinquent,
+		"grace_period_start":   s.analytics.FormatUnix(cus.GracePeriodStart),
+		"invoice_period_end":   s.analytics.FormatUnix(cus.InvoicePeriod.UnixEnd),
+		"invoice_period_start": s.analytics.FormatUnix(cus.InvoicePeriod.UnixStart),
+		"subscription_status":  cus.SubscriptionStatus,
+	}
+	if deps > 0 {
+		summary["dependents"] = deps
```

Comment:
allows updates to happen when deps not looked up, but wont overwrite dependents field from previous updates during cron